carball/json_parser/game_info.py

```diff
@@ -31,11 +31,13 @@ def parse_game_info_actor(self, actor_data: dict, game_event_actor: dict, object
         self.match_guid = actor_data.get('ProjectX.GRI_X:MatchGUID', '')
         self.playlist = actor_data['ProjectX.GRI_X:ReplicatedGamePlaylist']
         self.mutator_index = actor_data.get('ProjectX.GRI_X:ReplicatedGameMutatorIndex', 0)
-
-        if 'TAGame.GameEvent_Soccar_TA:SubRulesArchetype' in game_event_actor:
-            # Only used for rumble stats
-            # TODO can this contain any other mutators?
-            self.rumble_mutator = objects[game_event_actor['TAGame.GameEvent_Soccar_TA:SubRulesArchetype']]
-
+        try:
+            if 'TAGame.GameEvent_Soccar_TA:SubRulesArchetype' in game_event_actor:
+                # Only used for rumble stats
+                # TODO can this contain any other mutators?
+                self.rumble_mutator = objects[game_event_actor['TAGame.GameEvent_Soccar_TA:SubRulesArchetype']]
+        except TypeError:
```

Comment:
I'd rather check against NoneTypes than an ambiguous try catch

Reply:
Thanks for your review. You are probably right and I would be fine with an implementation like the one you suggested. The try catch was just my personal preference.